language-support/ts/daml-ledger/index.ts

```diff
@@ -586,7 +608,9 @@ class Ledger {
     template: Template<T, K, I>,
     key: K,
   ): Stream<T, K, I, CreateEvent<T, K, I> | null> {
+    let lastContractId: ContractId<T> | null = null;
     const request = [{templateId: template.templateId, key}];
+    const reconnectRequest = (): object[] => [{...request[0], 'contractIdAtOffset': lastContractId}]
```

Comment:
@hurryabit I'm a bit confused why this has to be lazy. I tried to define `reconnectRequest` as a variable or a function of the contractId, but it wouldn't change the `contractIdAtOffset` field.

Reply:
This can't be a variable since that would mean that you always use `null` for `lastContractId`, which is the value when we execute this line. What do you mean by "a function of the contract id" in this context?

Reply:
a function of type `contractId -> lastContractId`